docs/content/concepts/sui-architecture.mdx

```diff
@@ -16,11 +16,12 @@ Life of a Transaction details the transitions that all transactions on Sui go th
 
 Go to [Life of a Transaction](./sui-architecture/transaction-lifecycle.mdx).
 
-## Mempool and Consensus Engines
+## Consensus
 
-Narwhal and Bullshark combine to form the basis of the Sui mempool and consensus engine. This topic offers a brief introduction to the technologies that keep the activities on Sui synchronized and secure. 
+Zooming in, every transaction is sequenced by consensus, where validators agree to the same order of execution of the transactions, even if a minority of them are down or want to cause disruptions!
```

Comment:
`or want to cause disruptions!` , maybe we can explicitly mention here the malicious actors, so something like

"or they are malicious and want to cause disruptions"